app/models/user.rb

```diff
@@ -13,4 +13,8 @@ class User < ApplicationRecord
   def recent_posts
     posts.order(created_at: :desc).limit(3)
   end
+
+  def admin?
+    role == 'admin'
+  end
```

Comment:
- [ ] [**OPTIONAL** ] Kudos to implement the `user.admin?` method in the user model but, I'll suggest you implement it like this below so that we can have the ability to know if it's true or false.
```
def admin?
     role == 'admin' ? true : false
end

```

Reply:
thanks Sir @RolandM99 for your review. I will take care of implementing all the changes as requested